ecommerce_api_client/tests/test_auth.py

```diff
@@ -1,14 +1,20 @@
 # -*- coding: UTF-8 -*-
+
 from unittest import TestCase
+from mock import patch, Mock
```

Comment:
Imports from third-party libraries should be grouped and alphabetized after the Python imports.